docs/docker.md

```diff
@@ -4,9 +4,20 @@
 
 Using Docker is not required, but if you prefer using a Docker container we provide a Docker entrypoint to make this as easy as possible.
 
-We provide two distributions. The first is a based on Alpine Linux, while the second is based on Google's 
-["distroless"](https://github.com/GoogleContainerTools/distroless) debian12 images. 
+We provide images based on Alpine Linux and Google's 
+["distroless"](https://github.com/GoogleContainerTools/distroless) Debian12 images. 
 
+| Image              | Version                                                                                                             | Size                                                                                                                            | amd64 | armv7 | arm64v8 | i386 |
+|--------------------|---------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------|-------|-------|---------|------|
+| Distroless         | ![Docker Image Version](https://img.shields.io/docker/v/launchdarkly/ld-relay/latest-static-debian12-nonroot)       | ![Docker Image Size (tag)](https://img.shields.io/docker/image-size/launchdarkly/ld-relay/latest-static-debian12-nonroot)       | ✅     | ✅     | ✅       | ❌    |                                                                                                   |
+| Distroless (debug) | ![Docker Image Version](https://img.shields.io/docker/v/launchdarkly/ld-relay/latest-static-debian12-debug-nonroot) | ![Docker Image Size (tag)](https://img.shields.io/docker/image-size/launchdarkly/ld-relay/latest-static-debian12-debug-nonroot) | ✅     | ✅     | ✅       | ❌    |
+| Alpine             | ![Docker Image Version](https://img.shields.io/docker/v/launchdarkly/ld-relay/latest)                               | ![Docker Image Size (tag)](https://img.shields.io/docker/image-size/launchdarkly/ld-relay/latest)                               | ✅     | ✅     | ✅       | ✅    |
+
```

Comment:
TODO: these links just.. link to the badge image. Have them all link to the Dockerhub tags page, with a filter for the image tag in the URL.